TOC.md

```diff
@@ -496,6 +496,7 @@
   + [Migration FAQs](/faq/migration-tidb-faq.md)
 + [Glossary](/glossary.md)
 + Release Notes
+  + [TiDB V4.0 Roadmap](/roadmap.md)
```

Comment:
I think calling it v4.0 roadmap is a bit confusing in master, since once v4.0 has been released, it won't have new features added to it. There is a bug open on it: https://github.com/pingcap/docs/issues/2643 

Otherwise LGTM.

Reply:
The v4.0 roadmap in master will be replaced by the v5.0 roadmap, but since v5.0 roadmap has not been out yet, @scsldb and @uglyengineer  suggested we put the v4.0 version here for now. Please decide whether we keep this one in TOC or not and take a look at https://github.com/pingcap/docs/issues/2643. @scsldb

Reply:
FYI: I'll mark this file as draft according to https://github.com/pingcap/docs/issues/2643#issuecomment-666906822.
```suggestion
```